pkg/security/oauth/manager/manager.go

```diff
@@ -15,16 +15,16 @@ import (
 
 // Manager holds certificates from filesystem watched for changes
 type Manager struct {
-	mutex             sync.Mutex
-	config            clientcredentials.Config
-	requestTimeout    time.Duration
-	tickFrequency     time.Duration
-	startRefreshToken time.Time
-	token             *oauth2.Token
-	httpClient        *http.Client
-	tokenErr          error
-	doneWg            sync.WaitGroup
-	done              chan struct{}
+	mutex            sync.Mutex
+	config           clientcredentials.Config
+	requestTimeout   time.Duration
+	tickFrequency    time.Duration
+	nextRefreshToken time.Time
```

Comment:
Everything's OK, I would just rename this field - `nextRefreshToken` is hard to understand without context...

Reply:
rename `nextRefreshToken` to `nextTokenRenewalTime`